JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1253,4 +1275,45 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(descriptivesQQPlot)
 }
 
+.descriptivesPieChart <- function(dataset, options, variable) {
+  if (options$splitby != "" ) {
+    # return a collection
+    split <- names(dataset)
+
+    plotResult <- createJaspContainer(title=variable)
+    plotResult$dependOn(options="splitby", optionContainsValue=list(variables=variable))
```

Comment:
```suggestion
    plotResult$dependOn(optionContainsValue=list(variables=variable))
```
`options="splitby"` is already a dependency of the container.

Reply:
true, thanks for pointing it out